### PR TITLE
Add note about hashes

### PR DESCRIPTION
**Pull request by AGulev**

### CODE REVIEW

File: docs/en/manuals/script-properties.md [hunk 1]
```diff
@@ -16,7 +16,12 @@ Common use cases are to set the health or speed of a specific enemy AI, the tint
 
 ## Defining a script property
 
-Script properties are added to a script component by defining them with the `go.property()` special function. The function has to be used at the top level---outside any callback-functions like `init()` and `update()`. The default value provided for the property governs the type of the property: number, boolean, hash, `msg.url`, `vmath.vector3`, `vmath.vector4`, `vmath.quaternion` and `resource` (see below).
+Script properties are added to a script component by defining them with the `go.property()` special function. The function has to be used at the top level---outside any callback-functions like `init()` and `update()`. The default value provided for the property governs the type of the property: `number`, `boolean`, `hash`, `msg.url`, `vmath.vector3`, `vmath.vector4`, `vmath.quaternion` and `resource` (see below).
```
Comment (AGulev) [inline docs/en/manuals/script-properties.md:19]: Here I only changed formatting for boolean, number and hash. The actual change is the following block:

Comment (britzl) [inline docs/en/manuals/script-properties.md:19]: We call them lifecycle functions in other places of the documentation, not callback functions.

**Review by britzl — CHANGES_REQUESTED** [review]
See comment about callback vs lifecycle

File: docs/en/manuals/script-properties.md (hunk 1 above)
Comment (AGulev) [reply docs/en/manuals/script-properties.md:19]: haha, I didn't even see it because I didn't touch it, I'll fix

**Review by britzl — APPROVED** [review]

**Pull request merged**